### ISSUE TITLE
Multi-task loss

### ISSUE THREAD

**Issue by Sukaiqi** [swords123/IDA-3D#14]
I don't find the multi-task loss. Can you help me point out where their weights are set in code?
![loss](https://user-images.githubusercontent.com/46585700/125586057-a76dda8b-1455-424f-8527-288904f6dc86.jpg)

**Comment by swords123** [comment]
In the code, each loss is weighted after calculated, you can find these loss in [maskrcnn_benchmark/modeling/roi_heads/td_box_head_v3/loss.py](url) , [maskrcnn_benchmark/modeling/roi_heads/cost_volum_v11_head/loss.py](url)  and [maskrcnn_benchmark/modeling/rpn/stereo_rpn_v1/loss.py](url)

**Comment by Sukaiqi** [comment]
Thank you very much for your answer! I can find each loss in these files, but I can't find the weight. Can you help me point out the RPN weights(w1) as example in code? For that, I would be very grateful.

**Comment by swords123** [comment]
If I remember correctly， the w1 is 1.0 for RPN. For others, the weight is directly multiplied for each loss.
For example: 
rot_regression_loss = rot_regression_loss / rot_regression_target.numel() * 0.1

**Comment by Sukaiqi** [comment]
Thank you very much! I get it. I thought the weight was the same as Stereo R-CNN which is trained.
